schema/test/unit/tables/ciip_application_wizard_test.sql

```diff
@@ -74,6 +50,15 @@ select lives_ok(
     'ciip_administrator can insert data in ciip_application_wizard table'
 );
 
+select results_eq(
+  $$
+    select count(*) from ggircs_portal.ciip_application_wizard;
+  $$,
+  ARRAY[1::bigint],
```

Comment:
we started using the `values ( )` syntax for these tests since `ARRAY[]` has compatibility issues with some types - although the only issues we reported where either with boolean/text conversion and multiple types. So this one should be fine regardless?

```suggestion
$$
    values (1::bigint)
$$
```